cognite/client/data_classes/iam.py

```diff
@@ -35,7 +35,6 @@ def __init__(
 
 class ServiceAccountList(CogniteResourceList):
     _RESOURCE = ServiceAccount
-    _ASSERT_CLASSES = False
```

Comment:
why is this flag removed from all other classes as well?

Reply:
Not needed anymore.